src/bbsSignature.ts

```diff
@@ -128,7 +124,8 @@ export const createProof = async ({
     });
 
     return new Uint8Array(proof);
-  } catch {
+  } catch (e) {
+    console.error(e);
```

Comment:
Not sure we want to log to console. It's quite annoying if libraries do this under the hood